apps/web/src/modules/create-proposal/components/TransactionForm/Escrow/EscrowUtils.ts

```diff
@@ -0,0 +1,142 @@
+import { Address, encodeAbiParameters, toBytes, toHex } from 'viem'
+import { create } from 'zustand'
+import { createJSONStorage, persist } from 'zustand/middleware'
+
+import { EscrowFormState, EscrowFormValues } from './EscrowForm.schema'
+
+const KLEROS_ARBITRATION_PROVIDER =
```

Comment:
do you think we should pick up this address from `@smartinvoicexyz/*` packages? The rest looks great

Reply:
I am trying to avoid installing any package!